exporter/otlpexporter/otlp.go

```diff
@@ -132,43 +132,43 @@ func (e *exporter) shutdown(ctx context.Context) error {
 	return err
 }
 
-// getArrowStream returns the first-available stream when Arrow is
-// configured.  This returns nil when the consumer should fall back to
-// standard OTLP.
-func (e *exporter) getArrowStream(ctx context.Context) (*arrow.Stream, error) {
+// arrowSendAndWait gets an available stream and tries to send using
+// Arrow if it is configured.  A (false, nil) result indicates for the
+// caller to fall back to ordinary OTLP.
+func (e *exporter) arrowSendAndWait(ctx context.Context, data interface{}) (sent bool, _ error) {
```

Comment:
I like that we no longer return the stream and instead manage it entirely in the exporter. Overall feels cleaner here.

Reply:
Yeah, it makes the callers easier to reason about too.